test/bitcoin/bitcoinTestTools.mo

```diff
@@ -54,13 +54,13 @@ module {
     };
 
     // Returns the public key associated to `bitcoinPrivateKey`.
-    public func publicKey() : EcdsaTypes.PublicKey {
+    public func publicKey() : Blob {
       let publicPoint = Jacobi.toAffine(Jacobi.mulBase(bitcoinPrivateKey.key,
         Curves.secp256k1));
 
       return switch (PublicKey.decode(#point publicPoint)) {
         case (#ok publicKey) {
-          publicKey
+          Blob.fromArray(PublicKey.toSec1(publicKey, false).0)
```

Comment:
We should make `publicKey` match the [ecdsa_public_key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-ecdsa_public_key) signature 
> The return result is an extended public key consisting of an ECDSA public_key, encoded in [SEC1](https://www.secg.org/sec2-v2.pdf) **compressed form**

Reply:
If you are referring to whether it's compressed or not, why does it matter here? This is just to support the test data.

Reply:
Indeed you are right, it seems that [this line](https://github.com/tgalal/motoko-bitcoin/pull/12/commits/af8a6fad0c5711ec67fd3e48c64129886974ffce#diff-7e65ccae5995296041dc4031c3fb7b094ec0daecec4a621bd797014f07cb7e18R125) doesn't care about the compression aspect...